assets/css/structures/_table-of-content.scss

```diff
@@ -1,3 +1,9 @@
 .toc__list-item {
   break-inside: avoid;
+
+    a {
+    display: inline-block;
+    margin: 0.5rem 0.75rem;
```

Comment:
This `margin` seems to be adding some extra spacing around each linked word. Is this visual adjustment necessary for the red flags to not break?

If it's not necessarily, I'd prefer the alignment to be left-aligned without the extra padding.

_Left is original where all text is aligned left_; right is PR where there is extra spacing

<img width="516" alt="Side-by-side of [1] correctly rendered text (without a leading space) and [2] incorrectly rendered text (with a leading space)" src="https://user-images.githubusercontent.com/7484549/74808027-07133b00-533e-11ea-9e7d-c8a30c53a914.png">

Reply:
I think it’s fine to have `margin-left: 0` and go with the slightly indented flag. Which happens with non defined words, too.